src/components/SearchResultsPage/SearchResultsPage.js

```diff
@@ -44,6 +48,8 @@ const SearchResultPage = (props) => {
             values: ["Army", "Navy", "Air Force", "Marine Corps"]
```

Comment:
This will need to be configurable -- remember, OpenLXP operators are not necessarily DoD-specific.

Reply:
Yup, this whole section will be replaced by the dynamic filtering feature that I'm currently working on